src/adapters/git/super/actions/pull-request.ts

```diff
@@ -71,15 +63,16 @@ export class PullRequest extends GitSuper {
         branch: branchName,
         sha: fileSha,
       });
-      console.log("File updated in branch: ", branchName);
       this._context.logger.info(`File updated in branch: ${branchName}`);
 
-      // Create pull request
+      // Create pull request with initial body
       const { data: pr } = await this._context.octokit.rest.pulls.create({
         owner,
         repo,
-        title: `Update ${filePath}`,
-        body: "Automated update using autoedit",
+        title: `chore: update \`${filePath}\``,
+        body: `Update to \`${filePath}\` following this instruction:\n\n
```

Comment:
This is redundant because all the same information is in the title. As requested before you should do something like how I had it, which contained the original query and it also said who ran it. 

Given that you enabled support to run this from a GitHub comment, it might be nice to link back to the original comment as well.